example/otel-collector/main.go

```diff
@@ -117,18 +144,19 @@ func main() {
 		attribute.String("attrB", "raspberry"),
 		attribute.String("attrC", "vanilla"),
 	}
-
 	// work begins
 	ctx, span := tracer.Start(
 		ctx,
 		"CollectorExporter-Example",
 		trace.WithAttributes(commonAttrs...))
 	defer span.End()
+	meter, _ := metricglobal.Meter("test-meter").SyncInt64().Counter("test-counter")
```

Comment:
How about making this look less dummy than `test-counter`? This is counting iterations. So it could be named `loop-counter`?